packages/icons/package.json

```diff
@@ -33,7 +33,6 @@
     "@types/mock-fs": "4.13.1",
     "@types/node": "18.17.3",
     "@types/node-fetch": "2.6.4",
-    "@types/svgo": "2.6.4",
```

Comment:
This is deprecated and include in svgo 3: https://www.npmjs.com/package/@types/svgo